### DISCUSSION TITLE
Community Call Single Source/XML-Publishing with OJS

### DISCUSSION THREAD

**Discussion by ronste** [singlesourcepub/community#40] [Community Calls]
This is a regular call dedicated to discussions and exchange about how to integrate Single Source/XML-Publishing Workflows with OJS. Currently we meet every three months.

The topics of the calls centre around the different projects running or being developed in this field of activity, envisaged workflows as well as discussions of more detailed technical aspects.

The next meeting is scheduled  for:

- Thursday, 07.12.2023, 11:00 - 12:00 am, Meeting Room: https://fu-berlin.webex.com/fu-berlin/j.php?MTID=m9a845200a78acef92af74ab99b874963

Everyone is welcome to join. If you want to discuss specific topics please post a request here.

**Comment by xldrkp** [comment]
Thanks @ronste for posting your call here.
